src/mito2/src/sst/parquet/helper.rs

```diff
@@ -0,0 +1,86 @@
+// Copyright 2023 Greptime Team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use std::sync::Arc;
+
+use parquet::basic::ColumnOrder;
+use parquet::file::metadata::{FileMetaData, ParquetMetaData, RowGroupMetaData};
+use parquet::format;
+use parquet::schema::types::{from_thrift, SchemaDescriptor};
+use snafu::ResultExt;
+
+use crate::error;
+use crate::error::Result;
+
+// Refer to https://github.com/apache/arrow-rs/blob/7e134f4d277c0b62c27529fc15a4739de3ad0afd/parquet/src/file/footer.rs#L74-L90
+/// Convert [format::FileMetaData] to [ParquetMetaData]
+pub fn to_parquet_metadata(t_file_metadata: format::FileMetaData) -> Result<ParquetMetaData> {
```

Comment:
Parse? Since we have a `parse_column_orders()` fn
```suggestion
pub fn parse_parquet_metadata(t_file_metadata: format::FileMetaData) -> Result<ParquetMetaData> {
```